src/App/Pages/Vault/GroupingsPage/GroupingsPageViewModel.cs

```diff
@@ -87,7 +92,7 @@ public GroupingsPageViewModel()
         public bool HasCiphers { get; set; }
         public bool HasFolders { get; set; }
         public bool HasCollections { get; set; }
-        public bool ShowNoFolderCiphers => (NoFolderCiphers?.Count ?? int.MaxValue) < NoFolderListSize &&
+        public bool ShowNoFolderCipherGroup => (NoFolderCiphers?.Count ?? int.MaxValue) < NoFolderListSize &&
             (!Collections?.Any() ?? true);
```

Comment:
I think it's more readable like this:
```c#
public bool ShowNoFolderCipherGroup => NoFolderCiphers != null 
                                       && NoFolderCiphers.Count < NoFolderListSize 
                                       && Collections != null 
                                       && !Collections.Any();
```

Reply:
I agree this line is... _less_ readable (let's go ahead and say unreadable).  In fact I renamed the var after wasting a lot of time caused by a misinterpretation.  Unfortunately your modified version returns a different result, and I _think_ it's because of how the second condition is handled in the unmodified version (`?? true`).  In any case I left this one alone for fear of breaking it.  I can revisit when I'm thinking clearly again.